README.md

```diff
@@ -210,15 +210,15 @@ The Strings elements appearing in the above comment block are:
 * The `include` statement, which adds the usage example code.
 * Two `@param` tags, with the name of the parameter first, followed by a string describing the parameter's purpose.
 
-Puppet 4 is a typed language, so Puppet Strings automatically documents the parameter types from code. With Puppet 3, however, include the parameter type with the `@param` tag:
+Puppet 4 is a typed language, so Puppet Strings automatically documents the parameter types from code. You may optionally include a parameter type in the `@param` tag. Strings will emit a warning should the documented type differ from the actual type.
+
```

Comment:
Maybe worth mentioning that in this case it will default to what it gets from the code rather than the comment? Unless that's already mentioned elsewhere in the README